### ISSUE TITLE
Issue with importing uuid class?

### ISSUE THREAD

**Issue by psychohamster** [mechatroner/sublime_rainbow_csv#22]
Hello,
I've had a monthly batch process that I've been using for about a year based on this sublime plugin which has been working great. It uses some UDFs defined in the .rbql_init_source.py file that require the uuid functions. So i've had 'import uuid' at the top of the init file working great for ages. A few weeks ago I started trying to run the process (skipped Feb this year, so I think my last successful run was with 1.5.0) and now it acts like the import uuid statement no longer works, as I get 'undefined global uuid' error when running my function within the query. 
At first, I assumed it was something wonky with the sublime embedded python and started rolling back versions of ST3 to no avail. Even confirmed / recompiled the 3.3.6 pyo for the uuid class, etc. 
After no luck there, I tried running the same process using VSCode and its corresponding rbql plugin, with the same result. So, I'm guessing some sort of weird conflict has come up with uuid, since I can import sys and use functions there with no problem.

I confirmed I can exec the functions from the python cli on the system level.

**Steps for simple repro:**

Install any recent version of ST3 and install Rainbow_CSV from package control
 **.rbql_init_source.py**:
```
import uuid
import sys

def bob():
	return sys.version

def joe():
	return uuid.uuid4()
```

Open up a csv file in ST3 and run the following:
`update set a1 = joe() `

query execution, Error at line 1, details: global name 'uuid' is not defined

Now try this:
`update set a1 = bob()`
File executes and replaces with the expected embedded python version string

I'm going to downgrade to 1.5.0 and see if comes back, but figured I'd post here too in case anyone has run into this and has a quick solution I'm missing.

[updated] - downgraded the st3 package to 1.5.0 by replacing the folder under packages and now it works again. 

Thanks!

**Comment by mechatroner** [comment]
Hello @psychohamster,
Thank you very much for this bug report!
I spent half an hour in disbelief debugging this issue, turns out the root cause of the problem is this: https://stackoverflow.com/questions/12505047/why-doesnt-an-import-in-an-exec-in-a-function-work  - "import" does not work in "exec()" functions properly. The reason why "sys" module works is because it is imported by rbql itself. Starting from version 1.6.0 I switched to an updated rbql engine which instead of generating a temporary python module uses exec function. I think I had some tests with "import" statements in ".rbql_init_source.py", but they didn't fail because if you use imported modules without wrapping them into a function call - it works, e.g. this will work too: `update set a1 = uuid.uuid4()`. I will try to fix this problem ASAP.
